site/public/js/forum.js

```diff
@@ -345,6 +345,7 @@ function socketNewOrEditPostHandler(post_id, reply_level, post_box_id=null, edit
                 // eslint-disable-next-line no-undef
                 file_array[post_box_id] = [];
                 uploadImageAttachments(`#${post_id}-reply .upload_attachment_box`);
+                // eslint-disable-next-line no-undef
```

Comment:
It's better to list this function in the "global" section at the top of the file.  See more here: https://submitty.org/developer/software_and_system_design/coding_style_guide/javascript#style-rules